lib/travis/logs/receive.rb

```diff
@@ -1,49 +1,42 @@
 # frozen_string_literal: true
+
+require 'travis/exceptions'
 require 'travis/logs'
-require 'travis/support'
-require 'travis/amqp'
-require 'travis/support/exceptions/reporter'
-require 'travis/support/metrics'
+require 'travis/logs/helpers/database'
 require 'travis/logs/receive/queue'
 require 'travis/logs/services/process_log_part'
-require 'travis/logs/helpers/database'
 require 'travis/logs/sidekiq'
-require 'active_support/core_ext/logger'
+require 'travis/logs/sidekiq/log_parts'
+require 'travis/metrics'
 
 module Travis
   module Logs
     class Receive
```

Comment:
Looks like this is now the "drain"? Would it make sense to rename the class to `Drain` to make that more obvious?